crypto/src/main/java/org/web3j/crypto/WalletUtils.java

```diff
@@ -24,6 +26,7 @@
 public class WalletUtils {
 
     private static final ObjectMapper objectMapper = new ObjectMapper();
+    private static final SecureRandom secureRandom = new SecureRandom();
```

Comment:
Please get the SecureRandom instance from here https://github.com/web3j/web3j/blob/master/crypto/src/main/java/org/web3j/crypto/SecureRandomUtils.java

Reply:
Fixed.